modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -84,7 +86,7 @@ We add the [@deprecated](https://jsdoc.app/tags-deprecated.html) JSDoc tag to co
 in a future major release. This signals consumers to migrate to a more stable alternative before the
 deprecated code is removed.
 
-### FormField Main
+### Form Field Main
```

Comment:
Do we want this to be separated? Or should we keep this consistent with the component name (i.e. `FormField`)?

Reply:
So there is two possibilities:
- Have space between words
- Change `#form-field-main` to `#formfield-main`

`FormField` is a code name for component, and Form Field is what we use for storybook titles. I think it's okay to have them separated.

@mannycarrera4 what do you think?

Reply:
It looks as if in other Upgrade Guides we don't space it?

Reply:
I think for the title it should have a space, when referencing the component, no space but open to suggestions!

Reply:
We have spaces for component names in other Upgrade Guides. And we don't split utility names such as `useModel` etc. I think it's good keep consistent across all guides.